neural_compressor/adaptor/tf_utils/graph_converter.py

```diff
@@ -96,7 +96,7 @@ def __init__(self,
         """Convert graph.
 
         :param model: input tensorflow model.
-        :param qt_config: quantization configs, including interation and op-wise quant config
+        :param qt_config: quantization configs, including interaction and op-wise quant config
```

Comment:
Thanks for your feedback, it should be `iteration`.

Reply:
@chensuyue Thanks!